src/config/Config.ts

```diff
@@ -1,3 +1,10 @@
+declare global {
+	interface Window {
+		BACKEND_URL?: string;
+	}
+}
+
 export const BACKEND_URL = process.env.REACT_APP_BACKEND_URL ?? 'http://localhost:5000';
+window.BACKEND_URL = BACKEND_URL;
```

Comment:
Maybe something like this
```suggestion
export const DEV_DEPLOY = process.env.REACT_APP_DEPLOY === 'dev';
DEV_DEPLOY
	? // if DEV_DEPLOY assign BACKEND_URL to window.BACKEND_URL
	  (window.BACKEND_URL = BACKEND_URL)
	: // else make window.BACKEND_URL a getter to prevent it from being reassigned
	  Object.defineProperty(window, 'BACKEND_URL', { get: () => BACKEND_URL });
```